components/collections/Request.vue

```diff
@@ -14,7 +14,7 @@
           @click="!doc ? selectRequest() : {}"
           v-tooltip="!doc ? $t('use_request') : ''"
         >
-          <span :class="getRequestLabelColor(request.method)">{{ request.method }}</span>
+          <span>{{ request.method }}</span>
```

Comment:
```suggestion
          <span :class="getRequestLabelColor(request.method)">{{ request.method }}</span>
```
This shouldn't be removed for RESTful requests.